lib/nerves/package.ex

```diff
@@ -133,7 +127,6 @@ defmodule Nerves.Package do
     end
     platform = config[:platform]
     provider = provider(app, type)
-    compiler = config[:compiler]
```

Comment:
How do you feel about having a deprecation warning here to encourage everyone to remove this from their `nerves.exs` files?